pkg/apis/operator/register.go

```diff
@@ -37,7 +37,7 @@ var (
 		Group:    GroupName,
 		Resource: "knativeservings",
 	}
-	// KnativeServingResource respresents a Knative Serving
+	// KnativeEventingResource respresents a Knative Serving
```

Comment:
@yanggangtony you probably also want to change Knative Serving to Knative Eventing?